test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -60,3 +60,9 @@ def test_melspectrogram(self):
         transform = T.MelSpectrogram(sample_rate=sample_rate)
         waveform = get_whitenoise(sample_rate=sample_rate, duration=0.05, n_channels=2)
         self.assert_grad(transform, [waveform], nondet_tol=1e-10)
+
+    def test_mfcc(self):
```

Comment:
Note from offline discussion:

We should check for different values of `log_mel` as it changes the code path and operation used to compute MFCC.

https://github.com/pytorch/audio/blob/52decd2a4346f783a6229603716a08c2640aebe9/torchaudio/transforms.py#L558-L562